.github/workflows/pre-commit.yml

```diff
@@ -19,6 +19,10 @@ jobs:
     - uses: actions/setup-dotnet@v1
       with:
         dotnet-version: '3.1.x'
+    - name: Install dependencies
+      run: |
+          python -m pip install --upgrade pip
+          pip install pydoc-markdown
```

Comment:
This needs to be a fixed version in case the pydoc-markdown has a breaking update.

Reply:
Done.